src/xtgeo/xyz/polygons.py

```diff
@@ -312,31 +254,9 @@ def delete_columns(self, clist, strict=False):
             if cname in self._df:
                 self._df.drop(cname, axis=1, inplace=True)
 
+    @inherit_docstring(inherit_from=XYZ.from_file)
```

Comment:
> Yes now I recall why there is code duplication, as I did not find any obvious ways to keep the docstrings from the base class. Any solutions are welcomed.

My suggestion is doing this:
```
    @inherit_docstring(inherit_from=XYZ.from_file)
    def from_file(self, pfile, fformat="xyz"):
```
that means the user can do `help(Polygons.from_file)` and get the doc string from the base class.